main.go

```diff
@@ -89,7 +89,7 @@ func getUserAgent(appName string) string {
 
 func getConn(flags *flagStorage) (c *gcsx.Connection, err error) {
 	var tokenSrc oauth2.TokenSource
-	if flags.Endpoint.Hostname() == "storage.googleapis.com" {
+	if flags.CustomEndpoint.Hostname() == "storage.googleapis.com" {
```

Comment:
I think this `if` will not be entered by default because we are no longer keeping storage.googleapis.com. 
Anyways, we are deprecating jacobsa/gcloud. I would suggest you to merge your changes after that PR: https://github.com/GoogleCloudPlatform/gcsfuse/pull/1258

Reply:
Yeah, this change was just to ensure build success. I believe this function will not be exist after merging the mentioned PR. I'll wait for this.